### ISSUE TITLE
Annotators not able to access videos on web browser

### ISSUE THREAD

**Issue by himalimehta** [phuselab/DANTE#6]
Hi, @vcuculo , as you had said previously that the annotators will not need to install DANTE, and will only need to use their personalized link which is given after adding an annotator, along with a web browser. However these personal links are not giving the other annotators any access to the videos in order to annotate them. Only my personal link is working and I’m the only one who has this tool installed on my laptop. Can you please suggest how to fix this problem?

**Comment by vcuculo** [comment]
@himalimehta please be sure that you followed the instructions on the top page of the administration side:

<strong>Example</strong><br>
            To create two annotation groups, <em>alpha</em> and <em>beta</em>:
            <ol>
                <li>create the following directories <code>/video/alpha/</code> and <code>/video/beta/</code></li>
                <li>place the videos to be annotated in the relative directories.</li>
                <li>press <strong>Sync now</strong> button</li>
                <li>reload the page and add a new annotator, assigning the corresponding group</li>

**Comment by vcuculo** [comment]
You should have the annotators section as follows
![Schermata da 2021-03-25 16-39-05](https://user-images.githubusercontent.com/642555/112501231-5cafe780-8d89-11eb-96a7-51a0bfac3f90.png)

**Comment by himalimehta** [comment]
@vcuculo Yes I have tried doing this same thing. However even when I use my own link in a different laptop (with same browser) it shows a blank white page which says “Forbidden: You don’t have permission to access this resource.”

**Comment by vcuculo** [comment]
@himalimehta you need to install it on a server which is reachable from remote.

**Comment by himalimehta** [comment]
@vcuculo can you please explain how I can do that?

**Comment by vcuculo** [comment]
@himalimehta can you please describe your DANTE installation environment? where is it installed? how do you reach the web interface? which is the address do you use (public/private)?

**Comment by himalimehta** [comment]
@vcuculo dante files are installed on a windows laptop in the C:\wamp64\www folder. To sign in as an admin and reach the admin page, I use google chrome as my web browser. I’m not sure whether it’s public or private. I’m assuming it is private because I use localhost/index/.php?id=(my_id_number)
![image](https://user-images.githubusercontent.com/77561854/112509427-d4b8e600-8d66-11eb-8df7-78851041c0c9.jpg)

**Comment by vcuculo** [comment]
@himalimehta This answers your question. 
The annotators should reach your laptop via their browser, but this means that your laptop should be up and running 24/7 to let them annotate the videos whenever they prefer. DANTE should be installed on a public server to let others annotate your videos.

**Comment by himalimehta** [comment]
@vcuculo where should I move my files to a different location, so that it is installed on a “public server”?

**Comment by himalimehta** [comment]
@vcuculo also, I’m using a separate laptop and using google chrome to use my personal link and access the videos while simultaneously still running the laptop (connected to wamp server, and logged in as admin, and synced all videos) but the web browser on the new laptop still says “403 Forbidden: You don’t have permission to access this resource.” Is this because it’s not installed on a public server yet? Or should other annotators still be able to access the videos of the folder they are assigned to even if my laptop is running Dante on a private server?
